BA_yearly_subset.py

```diff
@@ -0,0 +1,117 @@
+import os
+import pandas as pd
+import klib
+import re
+
+data_dir = 'C:\\Users\\mcgr323\\projects\\tell\\inputs\\FERC_714\\'
```

Comment:
@mcgrathc is the functionality in this file to be included in the package?  If so, it would be good to do that at this point but watch out for naming function the same for different purposes.  E.g., `prepare_data`

Reply:
I will include this to the scripts directory and change it to match the match_fips_code.py